flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/JobAccumulatorsInfo.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.rest.messages;
+
+import org.apache.flink.runtime.rest.handler.job.JobAccumulatorsHandler;
+import org.apache.flink.util.Preconditions;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.annotation.Nullable;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Response type of the {@link JobAccumulatorsHandler}.
+ */
+public class JobAccumulatorsInfo implements ResponseBody {
+	public static final String FIELD_NAME_JOB_ACCUMULATORS = "job-accumulators";
+	public static final String FIELD_NAME_USER_TASK_ACCUMULATORS = "user-task-accumulators";
+
+	@JsonProperty(FIELD_NAME_JOB_ACCUMULATORS)
+	private List<JobAccumulator> jobAccumulators;
+
+	@JsonProperty(FIELD_NAME_USER_TASK_ACCUMULATORS)
+	private List<UserTaskAccumulator> userAccumulators;
+
+	@JsonCreator
+	public JobAccumulatorsInfo(
+		@JsonProperty(FIELD_NAME_JOB_ACCUMULATORS) @Nullable List<JobAccumulator> jobAccumulators,
```

Comment:
I would remove the `Nullable` and instead pass in an `Collections.emptyList()` if there are no job accumulators at the moment.

Reply:
make sense